front/src/page/News.tsx

```diff
@@ -55,11 +57,7 @@ export default () => {
       >
         {new_.title}
       </Header>
-      <div
-        dangerouslySetInnerHTML={{
-          __html: replace(new_.content, /\n/g, '<br />'),
-        }}
-      />
+      <ReactMarkdown children={new_.content} remarkPlugins={[remarkGfm]} />
```

Comment:
Here is the core code.

Reply:
Could you talk about the `remarkGfm` in your commit message? I am not sure what is it used for?